runtime/rastrace/trclog.c

```diff
@@ -2465,27 +2465,30 @@ static UtProcessorInfo * getProcessorInfo(void)
 	initHeader(&ret->procInfo.header, "PIN", sizeof(UtProcInfo));
 	osarch = j9sysinfo_get_CPU_architecture();
 	if (NULL != osarch) {
-		if ((strcmp(osarch, J9PORT_ARCH_PPC) == 0)
-		|| (strcmp(osarch, J9PORT_ARCH_PPC64) == 0)
-		|| (strcmp(osarch, J9PORT_ARCH_PPC64LE) == 0)
+		if ((strcmp(osarch, OMRPORT_ARCH_PPC) == 0)
+		|| (strcmp(osarch, OMRPORT_ARCH_PPC64) == 0)
+		|| (strcmp(osarch, OMRPORT_ARCH_PPC64LE) == 0)
```

Comment:
While you're changing these lines, please put the constants on the left (like line 2489):
```c
		if ((0 == strcmp(osarch, OMRPORT_ARCH_PPC)
		|| (0 == strcmp(osarch, OMRPORT_ARCH_PPC64)
		|| (0 == strcmp(osarch, OMRPORT_ARCH_PPC64LE)
```

Reply:
Changed as requested.